src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -605,6 +606,24 @@ public static Index Open(string path, DocumentDatabase documentDatabase, bool ge
             }
         }
 
+        private static void AttemptToLinkDatabaseAndIndexJournals(string name, StorageEnvironmentOptions indexOptions, DocumentDatabase documentDatabase)
+        {
+            if (documentDatabase.Configuration.Storage.AvoidSharedJournals is false &&
+                indexOptions.CanJournalsBeLinkedWith(documentDatabase.DocumentsStorage.Environment.Options))
+            {
+                // here we enable the root / branch model for this index
+                documentDatabase.IndexStore.RegisterSharedJournals(indexOptions);
+                return;
+            }
+
+            var logger = documentDatabase.Loggers.GetLogger(typeof(Index));
```

Comment:
this is incorrect, we need to implement 'GetLoggerForIndex' (check impl in raven log manager extensions) for DatabaseLoggersContext